src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (numberOne) => {
+    const result = (numbertwo) => {
+      const sum = numberOne + numbertwo;
+
+      if (counter > 3) {
+        return counter % 2 === 0
```

Comment:
If following the comment above, here will be better to move the expression `counter % 2 === 0` to `if` statement.
```suggestion
      if (counter > 3 && counter % 2 === 0) {
        return 'Bzzz... Error!';
```

Reply:
I had no thinking that it's possible. Now it's fixed.